tests/multinode.rs

```diff
@@ -464,6 +464,10 @@ fn test_multi_node_dynamic_network() {
         }
     }
     assert_eq!(consecutive_success, 10);
+    for (_, node) in &validators {
+        node.exit();
+    }
+    server.exit();
     for (_, node) in validators {
         node.close().unwrap();
```

Comment:
`close()` -> `join()`

Reply:
Same for `server.close()`

Reply:
should `Service::join` call `FullNode::join`?  I notice that  `Service::join` is not calling exit.